### ISSUE TITLE
Adding `_total` to `Metric` doesn't offset the `_total` if `_total` already exist

### ISSUE THREAD

**Issue by vivekpatani** [prometheus/client_python#912]
Here's the sample code
```python3
metric = Metric('num_requests_total', 'Total Number of Requests' , 'counter')
metric.add_sample('num_requests_total', value=stat['num_requests_total'], labels={'xyz': stat['name']}
```

This shows up on the prometheus endpoint as
```
# TYPE num_requests_total_total counter
num_requests_total{xyz="random_name"} 10.0
```

Now there are two `_total_total` at the end. Am I missing something here? I see the `counterMetric` function already has logic to remove the extra `_total` towards the end.

Thanks.

**Comment by csmarchbanks** [comment]
This looks like a bug to me, the TYPE should also just be `num_requests_total` I believe.

**Comment by mansoor17syed** [comment]
Hi,
I would like contribute here.
Please assign to me.

Thank You

**Comment by vivekpatani** [comment]
any updates on this? Thanks.

**Comment by mansoor17syed** [comment]
Hi
https://github.com/anantha96
Shall push the changes

**Comment by anantha96** [comment]
Hi @csmarchbanks, Fix submitted in #951

**Comment by csmarchbanks** [comment]
While reviewing #951 I realized that you are using an internal only method and we already have the appropriate logic in `Counter` and `CounterMetricFamily`. 

You will see in the docstring for `Metric` that:
```
This is intended only for internal use by the instrumentation client.
```

I am going to close this issue as you should be using either `Counter` or `CounterMetricFamily` instead of `Metric`.
